src/xava.c

```diff
@@ -224,6 +224,22 @@ as of 0.4.0 all options are specified in config file, see in '/home/username/.co
         thing.xava = &xava;
         xavaIONotifyAddWatch(&thing);
 
+        // attach pywal colors file to the IONotify thing
+        static const char b[] = "/.cache/wal/colors";
```

Comment:
Multiple things need to be fixed:
 - I'd like to comply with the [XDG base specification](https://wiki.archlinux.org/title/XDG_Base_Directory), this means that we should NEVER point to a specific folder, but rather let that depend on the environment variables and if they don't exist only THEN grab the hardcoded value. Since this isn't completely the first time this is done, I'd like this to be done within the src/shared family of functions (I'll do these myself if you don't want to make them)
 - pywal support should be optional, the current implementation will fail if the user doesn't happen to use pywal. Consider users of Windows or MacOS, this will straight up fail for them